Production/python/Production.py

```diff
@@ -85,13 +85,21 @@
     process.combinatoricRecoTaus.builders[0].signalConeSize = cms.string('max(min(0.2, 4.528/(pt()^0.8982)), 0.03)') ## change to quantile 0.95
     process.selectedPatTaus.cut = cms.string('pt > 18.')   ## remove DMFinding filter (was pt > 18. && tauID(\'decayModeFindingNewDMs\')> 0.5)
 
-tauIdConfig = importlib.import_module('TauMLTools.Production.runTauIdMVA')
+tauIdConfig = importlib.import_module('RecoTauTag.RecoTau.tools.runTauIdMVA')
 updatedTauName = "slimmedTausNewID"
 tauIdEmbedder = tauIdConfig.TauIDEmbedder(
-    process, cms, debug = False, updatedTauName = updatedTauName,
-    toKeep = [ "2017v2", "dR0p32017v2", "newDM2017v2", "deepTau2017v2p1"]
+    process, cms, updatedTauName = updatedTauName,
+    toKeep = [ "2017v2", "dR0p32017v2", "newDM2017v2", "deepTau2017v2p1", "newDMPhase2v1"]
 )
-tauIdEmbedder.runTauID(tau_collection = tau_collection)
+tauIdEmbedder.runTauID()
```

Comment:
The official CMSSW version of `runTauIdMVA` does not have a possibility to modify the tau collection. Should we ask for a corresponding change in official CMSSW rather than using it the TauML specific one? You can find the (very nasty & not that bulletproof) workaround below...

Reply:
solved